release/nightly_tests/dataset/multi_node_train_benchmark.py

```diff
@@ -169,6 +184,12 @@ def train_loop_per_worker():
         it = train.get_dataset_shard("train")
     device = train.torch.get_device()
 
+    # Setup the model
+    raw_model = resnet50(weights=ResNet50_Weights.DEFAULT)
```

Comment:
Won't this use the pretrained weights?

https://pytorch.org/vision/main/models/generated/torchvision.models.resnet50.html#torchvision.models.ResNet50_Weights